app/subscriber/src/components/content-list/ContentReportPin.tsx

```diff
@@ -34,11 +34,16 @@ export const ContentReportPin = ({ contentId }: IContentReportPinProps) => {
     );
   }, [contentId, myReports, reportContent]);
 
-  if (!reports.length) return <></>;
+  if (!reports.length) return <div className="icon-placeholder" />;
 
   return (
     <styled.ContentReportPin className="content-report-pin">
-      <FaThumbtack data-tooltip-id={`report-pin-${contentId}`} />
+      <img
```

Comment:
Why are we switching to an image instead of the icon?

Reply:
SVG provided by Bobbi

Reply:
(prob doesn't matter if I leave it as you had it)